packages/compass-sidebar/src/components-legacy/sidebar/sidebar.module.less

```diff
@@ -22,6 +20,24 @@
   position: relative;
   z-index: 50;
 
+  --color: @pw;
+  --bg-color: @compass-sidebar-base-background-color;
+
+  --icon-color: @pw;
```

Comment:
☝️ temporarily including the new css vars on the old sidebar as well so the changes in compass-databases-navigation can pick it up.